plugins/CoreHome/stylesheets/layout.less

```diff
@@ -148,13 +155,16 @@ nav {
   }
 
   @media all and (max-width: 992px) {
+    #secondNavBar {
+      width: 0 !important;
+    }
+
     .pageWrap {
-      margin-left: 0 !important;
+      width: 100% !important;
```

Comment:
```suggestion
      width: calc(~'100% - 18px') !important; // 100% - pageWrap margin
```

Or we need to replace the margin with more padding for those resolutions. Otherwise the content will be stretched to the scrollbar (and create a horizontal scrollbar):

![](https://github.com/matomo-org/matomo/assets/635801/9a6f1309-082a-442d-82a8-6d6f0c41f30e)

It should look like this with the change:

![](https://github.com/matomo-org/matomo/assets/635801/3eaf19e9-5be8-4d44-91ec-f2087593342a)

_Yes, the navigation elements are already misaligned with their custom margins. That was not changed here._

Reply:
Both Firefox 120 and Chrome 120.

Just a caveat: the visual problem only occurs if the datatable (the screenshots are from "Behaviour > Pages") has content. If it is empty it does not stretch to the scrollbar (but also should not begin to do that with the change).

Reply:
Ah. Yes. There it is reproducible.